README.md

```diff
@@ -1,222 +1,515 @@
+The style guide must be updated to reflect all changes.
+
+### Linting
+
+Much of this guide can be enforced by using a linter such as [JSHint](https://github.com/jshint/jshint).
+The provided [`.jshintrc`](https://github.com/yola/jsgreat/blob/master/.jshintrc) file should be used to help maintain conformity.
+
+### Installation
+
+Install via [npm](https://www.npmjs.com/package/jsgreat):
+
+```bash
+$ npm install jsgreat --save-dev
+```
+
+Extend the provided `.jshintrc` file:
+
+```json
+{
+  "extends": "./node_modules/jsgreat/.jshintrc"
+}
+```
+
+## Rules
 
-Many of these options will be enforced by [this .jshintrc file](https://github.com/yola/wiki/blob/master/Files/.jshintrc). This file can be used by tools that support JSHint integration. For most projects, it is strongly recommended to use Grunt and the grunt-contrib-jshint plugin to run jshint.
+Serving as a base, the [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml) should be referenced for topics not addressed below.
 
-The following additions and variations will be enforced:
+### Naming
 
-## Strings
-The guide [prefers `'` over `"`](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Strings#Strings). We only require that quotes are used consistently within the file. This will be [enforced by jshint](http://www.jshint.com/docs/options/#quotmark).
+* Variables, constants, enums, methods, and properties should be written in `lowerCamelCase`.
+* Classes and global namespaces should be written in `UpperCamelCase`.
+* File names should be written in `hyphenated-lowercase.js`.
 
-## Indentation
-Prefer 2-space indentation for all Javascript and JSON files.
+### Variables
 
-## Multiple Vars
-Prefer multiple `var` statements, each on their own line, for any variables that have assignments. Multiple variables without assignments can be `var`ed together on a single line.
+* When available, the `let` keyword should be used instead of `var`.
+
+#### Variable statements
+
+* Variable statements with multiple assignments should not be used.
+* Each variable assignment should exist on its own line.
+* Multiple variables without assignments can be declared on a single line.
 
 **Don't do this:**
+
 ```javascript
-var foo = 'foo',
+let foo = 'foo',
   bar = 'bar',
   baz = 'baz';
 ```
+
 **Do this instead:**
+
 ```javascript
-var foo = 'foo';
-var bar = 'bar';
-var baz = 'baz';
+let foo = 'foo';
+let bar = 'bar';
+let baz = 'baz';
 ```
+
 **This is okay, too:**
+
 ```javascript
-var foo, bar, baz;
+let foo, bar, baz;
 ```
 
-## Function Declarations vs. Function Expressions
+#### Unused Variables
+
+Unused variables should be removed from the code when they are no longer needed.
 
-Prefer `var`ed function expressions over function declarations.
+Although discouraged, unused function parameters are allowed. This is due to some libraries expecting specific function signatures.
```

Comment:
Unused function parameters should be removed, unless necessary to maintain compatibility with other libraries. ?